lib/resources/r4/communication.yaml

```diff
@@ -134,20 +117,18 @@ fields:
 - name: recipient
   required: 'Yes'
   cardinality: 1..*
-  type: Reference(Practitioner/Group)
-  description: The Practitioner or Pool (Group) that the communication is intended to reach
```

Comment:
pick 'Communication' or 'communication' to use in descriptions throughout the file

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/720c35bf50e9cb8dd953ad8a2bce0b6a3c29f74a#